gnupg.yaml

```diff
@@ -0,0 +1,228 @@
+package:
+  name: gnupg
+  version: 2.2.41
+  epoch: 0
+  description: GNU Privacy Guard 2 - meta package for full GnuPG suite
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: GPL-3.0-or-later
+secfixes:
+  2.2.41-r0:
+    - CVE-2022-34903
+    - CVE-2020-25125
+    - CVE-2019-14855
+    - CVE-2018-12020
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - bzip2-dev
+      - gettext-dev
+      - libassuan-dev
+      - libgcrypt-dev
+      - libgpg-error-dev
+      - libksba-dev
+      - libtool
+      - libusb-dev
+      - npth-dev
+      - openldap-dev
+      - sqlite-dev
+      - texinfo
+      - zlib-dev
+      - gnutls-dev
+      - glibc-iconv
+      # - pinentry TODO skipping for now as seems to require gtk+3.0, gcr, libproxy and mesa
```

Comment:
I skipped this dependency as it was starting to open a can of worms.  Everything's compiling without it, so maybe we add later when needed?